src/patternfly/components/InlineEdit/examples/InlineEdit.md

```diff
@@ -0,0 +1,362 @@
+          {{/inline-edit-value}}
+          {{#> inline-edit-group inline-edit-group--modifier="pf-m-input-group pf-m-column"}}
+            {{#> inline-edit-input}}
+              {{#> check}}
+                {{#> check-input check-input--attribute=(concat 'id="' inline-edit--id inline-edit--row 'check-1" name="' inline-edit--id inline-edit--row 'example-check"')}}{{/check-input}}
+                {{#> check-label check-label--attribute=(concat 'for="' inline-edit--id inline-edit--row 'check-1"')}}Check 1{{/check-label}}
+              {{/check}}
+            {{/inline-edit-input}}
+            {{#> inline-edit-input}}
+              {{#> check}}
+                {{#> check-input check-input--attribute=(concat 'id="' inline-edit--id inline-edit--row 'check-2" name="' inline-edit--id inline-edit--row 'example-check-2"')}}{{/check-input}}
+                {{#> check-label check-label--attribute=(concat 'for="' inline-edit--id inline-edit--row 'check-2"')}}Check 2{{/check-label}}
+              {{/check}}
+            {{/inline-edit-input}}
+          {{/inline-edit-group}}
+        {{/table-td}}
+        {{#> table-td table-td--data-label="Radios"}}
+          {{#> inline-edit-value}}
+            Radio 1, Radio 2
+          {{/inline-edit-value}}
+          {{#> inline-edit-group inline-edit-group--attribute='role="radiogroup" aria-label="Radio group example"' inline-edit-group--modifier="pf-m-input-group pf-m-column"}}
+            {{#> inline-edit-input}}
+              {{#> radio}}
+                {{#> radio-input radio-input--attribute=(concat 'id="' inline-edit--id inline-edit--row 'radio-1" name="' inline-edit--id inline-edit--row 'example-radio"')}}{{/radio-input}}
+                {{#> radio-label radio-label--attribute=(concat 'for="' inline-edit--id inline-edit--row 'radio-1"')}}Radio 1{{/radio-label}}
+              {{/radio}}
+            {{/inline-edit-input}}
+            {{#> inline-edit-input}}
+              {{#> radio}}
+                {{#> radio-input radio-input--attribute=(concat 'id="' inline-edit--id inline-edit--row 'radio-2" name="' inline-edit--id inline-edit--row 'example-radio"')}}{{/radio-input}}
+                {{#> radio-label radio-label--attribute=(concat 'for="' inline-edit--id inline-edit--row 'radio-2"')}}Radio 2{{/radio-label}}
+              {{/radio}}
+            {{/inline-edit-input}}
+          {{/inline-edit-group}}
+        {{/table-td}}
+        {{#> table-td table-td--data-label="Number"}}
+          {{#> inline-edit-value}}
+            2
+          {{/inline-edit-value}}
+          {{#> inline-edit-input}}
+            {{> form-control controlType="input" input="true" form-control--attribute=(concat 'type="number" value="2" id="' inline-edit--id inline-edit--row 'number-input"  aria-label="Number input"')}}
+          {{/inline-edit-input}}
+        {{/table-td}}
+        {{#> table-td table-td--action="true"}}
+          {{#> inline-edit-group inline-edit-group--modifier="pf-m-action-group"}}
```

Comment:
Will have to add in styles to vertically center these actions

<img width="451" alt="Screen Shot 2020-01-07 at 4 06 49 PM" src="https://user-images.githubusercontent.com/20118816/71929607-f9c43780-3167-11ea-8569-833d6bbd84ee.png">